content/en/dashboards/ddsql_editor/reference/statements.md

```diff
@@ -0,0 +1,398 @@
+
+`condition`
+: An expression that is evaluated and interpreted implicitly as having a boolean result.
+
+`expression`
+: A value expression. See [Expressions](#expressions) for details and examples.
+
+### Evaluation
+
+SELECT retrieves rows from zero or more tables. The general processing of SELECT is as follows:
+
+1. All elements in `FROM` are computed. If more than one element is specified, they are joined together using the specified join type.
+2. If the `WHERE` clause is specified, rows that do not satisfy the condition are eliminated from the output.
+3. If the `GROUP BY` clause is specified or there are aggregate function calls in the `selectExpr`, the output is combined into groups of rows that match on one or more values, and the aggregates are computed. If `HAVING` is present, rows that do not satisfy its condition are eliminated from the output.
+4. The actual output rows are computed using the `selectExpr`.
+5. `SELECT DISTINCT` eliminates duplicate rows from the result.
+6. If the `ORDER BY` clause is specified, the returned rows are sorted in the specified order.
+7. If the `LIMIT` or `OFFSET` clause is specified, rows not in the specified subset are eliminated.
+
+The system may execute the query in any way that is guaranteed to produce the results specified by this order.
+
+## Aliases
+
+Aliases are substitute names for output expressions or `FROM` items. An alias is used for brevity or to eliminate ambiguity for self-joins (where the same table is scanned multiple times).
+
+{{< code-block lang="sql" >}}
+SELECT * FROM my_long_hosts_table_name as hosts
+{{< /code-block >}}
+
+When an alias is provided in a `FROM` item, it completely hides the actual name of the table or function. In the above example, the remainder of the DQL expression must refer to `my_long_hosts_table_name` as `hosts`.
+
+## Ordinals
+
+`GROUP BY` and `ORDER BY` clause expressions can be column names, arbitrary expressions formed from input columns, or the name or ordinal number of an output expression (a `SELECT` expression). Output expression ordinals are 1-indexed.
+
+For example, the output of this query is ordered first by `ex1`, then `ex2`, and then `ex3`:
+
+{{< code-block lang="sql" >}}
+SELECT ex1, ex2, ex3 FROM table ORDER BY 3, 2, 1;
+{{< /code-block >}}
+
+## AGGR
+
+`AGGR` is a DDSQL statement type that operates on metrics data. Its primary goal is to make executing timeseries queries more natural, flexible, composable, and concise.
+
+<!-- QUERY: Should the syntax template below include INTERPOLATE? -->
+
+### Syntax
```

Comment:
Should this query template include `INTERPOLATE`?

Reply:
Yes please

Reply:
@dd-annarose I'm not sure of the exact syntax, but if you could make a suggestion on the PR to add it, happy to accept the suggestion!